.github/workflows/build.yml

```diff
@@ -309,6 +309,13 @@ jobs:
           ./admin/pull-readme
           git diff --exit-code
 
+      - name: Spell check code
+        uses: crate-ci/typos@85f62a8a84f939ae994ab3763f01a0296d61a7ee
```

Comment:
AFAICT this is the first `uses:` that pins to an explicit rev instead of a tag. Were you thinking we should do that going forward, or is there more to this that justifies a comment?